tests/test_other.py

```diff
@@ -8051,11 +8051,12 @@ def test(args, closure, opt):
         test(['-s', 'WASM=0'], closure, opt)
         test(['-s', 'WASM_ASYNC_COMPILATION=0'], closure, opt)
 
-  # TODO: Debug why the code size is different on Windows. Also, for some
-  # unknown reason (at time of writing), this test is not skipped on the Windows
-  # autoroller, despite the bot being correctly configured to skip this test.
-  # The no_windows decorator also solves that problem.
+  # TODO: Debug why the code size is different on Windows and Mac. Also, for
+  # some unknown reason (at time of writing), this test is not skipped on the
+  # Windows autoroller, despite the bot being correctly configured to skip this
```

Comment:
Is this configured to be skipped on Mac autoroller too? (Where do we configure those things?)

Reply:
Yes, it is configured in [bots.json](https://chromium.googlesource.com/emscripten-releases/+/refs/heads/master/bots.json). It seems the whole skipping mechanism has some serious problems.

Reply:
It seems it is configured to be skipped in every platform, including linux, right? Maybe I should fix the comment.